src/Lock - Closed.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
So this and `Local - Closed - Fill.svg` should be the same, correct?

Reply:
Yes they are